archetypes/mp-bare/src/main/resources/archetype-resources/src/main/resources/logging.properties.vm

```diff
@@ -0,0 +1,25 @@
+# Example Logging Configuration File
+# For more information see $JAVA_HOME/jre/lib/logging.properties
+
+# Send messages to the console
+handlers=java.util.logging.ConsoleHandler
+
+# Global default logging level. Can be overriden by specific handlers and loggers
+.level=INFO
+
+# Helidon Web Server has a custom log formatter that extends SimpleFormatter.
+# It replaces "!thread!" with the current thread name
+java.util.logging.ConsoleHandler.level=INFO
```

Comment:
This is of course correct and fine, but I have found that `java.util.logging` rookies don't realize they have to set this to, say, `FINER` to see log messages that are logged at, say, `FINER`, `FINE`, etc.  I usually default mine to `FINER` and then happily forget about it.

Reply:
That was also copied from the helidon quickstarts, not sure what to do here.